### PR TITLE
Feat/pickup options

### PR DESCRIPTION
**Pull request by leoWorkingGood**
#### What problem is this solving?

- Recieve in type Shipping the pickup options (pickupOptions)
- Mutation to allow to select a pickup point of a product in the product page

Closes #94 

#### How should this be manually tested?

Here it is a print from graphql-ide showing the mutation working, and the pickupOptions type showing all the pickup pointer available    https://prnt.sc/
The mutation from print: [Link](https://vtexpr--iocea.myvtex.com/_v/private/vtex.checkout-graphql@0.42.0/graphiql/v1?operationName=pick&query=mutation%20pick(%24id%3A%20ID%2C%20%24pickID%3A%20String)%20%7B%0A%20%20selectPickupOption(orderFormId%3A%20%24id%2C%20pickupOptionId%3A%20%24pickID)%20%7B%0A%20%20%20%20id%0A%20%20%20%20shipping%20%7B%0A%20%20%20%20%20%20pickupOptions%20%7B%0A%20%20%20%20%20%20%20%20id%0A%20%20%20%20%20%20%20%20price%0A%20%20%20%20%20%20%20%20isSelected%0A%20%20%20%20%20%20%20%20friendlyName%0A%09%09%09%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A&variables=%7B%0A%20%20%22id%22%3A%20%221321189263234cff8f4848dcc1c7b6ec%22%2C%0A%20%20%22pickID%22%3A%20%22Retirada%20na%20Loja%20(SMO)%22%0A%7D)

The product used [prod](https://vtexpr--iocea.myvtex.com/blusa-feminina-basica-texturizada-manga-bufante-decote-redondo-preta-9856461-preto/p)

SKU of itens: 2676332 and 2676333

#### Checklist/Reminders

- [*] Updated `README.md`.
- [*] Updated `CHANGELOG.md`.
- [*] Linked this PR to a Jira story (if applicable).
- [*] Updated/created tests (important for bug fixes).
- [*] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements 

#### Notes

Lucas Cordeiro and Bruno Moreira are directly helping in this MR approval

### CODE REVIEW

**Review by lucasecdb — DISMISSED** [review]

File: node/constants/index.ts [hunk 1]
```diff
@@ -9,6 +9,6 @@ export enum AddressType {
 
 // Delivery channels
 export const DELIVERY = 'delivery'
-export const PICKUP_IN_STORE = 'pickup-in-store'
+export const PICKUP_IN_STORE = 'pickup-in-point'
```
Comment (darlenedms) [inline node/constants/index.ts:12]: I would suggest to change the const name too.
```suggestion
export const PICKUP_IN_POINT = 'pickup-in-point'
```

**Review by darlenedms — CHANGES_REQUESTED** [review]
Just rename the const name and you are good to go. 😄

**Comment by leoWorkingGood** [comment]
@darlenedms updated =)

**Comment by lucasecdb** [comment]
@leoWorkingGood you'll need to update it in all the files that use this constant

**Comment by leoWorkingGood** [comment]
@lucasecdb All the files were updated with the new constant name

**Review by lucasecdb — APPROVED** [review]

**Review by darlenedms — APPROVED** [review]
🚚

**Pull request merged**